tests_zemu/tests/masp.test.ts

```diff
@@ -167,4 +167,33 @@ describe('Masp', function () {
       await sim.close()
     }
   })
+
+  test.concurrent.each(MASP_MODELS)('Clean randomness Buffers', async function (m) {
+    const sim = new Zemu(m.path)
+    try {
+      await sim.start({ ...defaultOptions, model: m.name })
+      const app = new NamadaApp(sim.getTransport())
+
+      const respSpend = await app.getSpendRandomness()
+      console.log(respSpend)
+      expect(respSpend.returnCode).toEqual(0x9000)
+      expect(respSpend.errorMessage).toEqual('No errors')
+
+      const respOutput = await app.getOutputRandomness()
+      console.log(respOutput)
+      expect(respOutput.returnCode).toEqual(0x9000)
+      expect(respOutput.errorMessage).toEqual('No errors')
+
+      const respRandomness = await app.getConvertRandomness()
+      console.log(respRandomness)
+      expect(respRandomness.returnCode).toEqual(0x9000)
+      expect(respRandomness.errorMessage).toEqual('No errors')
+
+      const respClean = await app.cleanRandomnessBuffers()
+      console.log(respClean)
+      expect(respClean.returnCode).toEqual(0x9000)
```

Comment:
How do we know with this test that the buffer was cleaned? After cleaning the buffer, should we initiate a new transaction or retrieve some value to ensure that the buffer is indeed cleaned?

Reply:
I think we do not have a way of checking that because this takes action directly into the nv memory. But I added the test to make sure at least that the method is not crashing anywhere.